hawc/main/settings/staging.py

```diff
@@ -43,9 +43,9 @@
 else:
     raise ValueError(f"Unknown email backend: {email_backend}")
 
-LOGGING["loggers"]["django"]["handlers"] = ["file"]
-LOGGING["loggers"]["hawc"]["handlers"] = ["file"]
-LOGGING["loggers"]["hawc.request"]["handlers"] = ["hawc-request"]
+LOGGING["loggers"]["django"]["handlers"] = ["console", "file"]
+LOGGING["loggers"]["hawc"]["handlers"] = ["console", "file"]
+LOGGING["loggers"]["hawc.request"]["handlers"] = ["console", "hawc-request"]
 
```

Comment:
Do you not have to add a handler for your new `hawc.audit.change` logger?

Reply:
Nope; it's handled by the "hawc" logger as a parent.

Reply:
Didn't find a great link in the python docs... see https://khashtamov.com/en/ultimate-guide-to-python-logging/

> Loggers can inherit logging settings from their ancestors. The inheritance is defined by the . in logger's name. If you have the logger called my_package.logger1 it will inherit the settings defined for my_package.

Reply:
Ahh makes sense!